.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_size/aws_cryptosdk_enc_ctx_size_harness.c

```diff
@@ -31,22 +31,27 @@ void hash_iterator_generator(struct aws_hash_iter *new_iter, const struct aws_ha
 void hash_iterator_generator2(struct aws_hash_iter *new_iter, const struct aws_hash_iter *old_iter) {
     (void)old_iter;
     if (new_iter->status == AWS_HASH_ITER_STATUS_READY_FOR_USE) {
-        new_iter->element.key   = malloc(sizeof(struct aws_string));
+        new_iter->element.key = malloc(sizeof(struct aws_string));
+        __CPROVER_assume(new_iter->element.key != NULL);
         new_iter->element.value = malloc(sizeof(struct aws_string));
+        __CPROVER_assume(new_iter->element.value != NULL);
     }
 }
 
 void aws_cryptosdk_enc_ctx_size_harness() {
+    /* Nondet Input */
     struct aws_hash_table map;
+    size_t size;
```

Comment:
It would be clearer to malloc these, then assume nonnull.

Reply:
Updated!